.buildkite/core.rayci.yml

```diff
@@ -315,3 +315,14 @@ steps:
       - forge
       - raycpubase
       - corebuild
+
+  - label: ":ray: core: multi gpu tests"
+    tags:
+      - python
```

Comment:
change to `accelerated_dag`; `python` will catch most of prs

Reply:
Btw, what exactly are the tags here vs the ones in BUILD files? Are they the same tags?

Reply:
yess, `tags` is a custom feature we build on top of the normal buildkite yaml definition; these tags determine whether a buildkite step is run; it's different from the ones in BUILD files, which is the tag for bazel test target

the file ci/pipeline/determine_tests_to_run.py, if you run it locally

> python ci/pipeline/determine_tests_to_run.py --rayci_tags

will look at your PR, changes, etc. and produce a list of tags such as

> rllib data python accelerated_dag ...

only the buildkite step has one of those tags will run; otherwise it will be skipped

Reply:
I see, thanks!